ui/widgets/nuxeo-user-tag.js

```diff
@@ -56,6 +56,13 @@ import './nuxeo-tooltip.js';
           a:hover {
             @apply --nuxeo-link-hover;
           }
+
+          .user-tag-ellipsis {
+            white-space: nowrap;
+            max-width: 80px;
```

Comment:
can you clarify why are we using `80px` here? Where is this number coming from? We should avoid magic numbers because our clients configure our solution and we need to account for that.

Reply:
80 px is an appropriate width of the table cell to put ellipsis for the user tag text, contained in an oval tag. If the width is increased further the shape of the tag becomes rectangular.